Dynatrace/dynatrace-operator
# Update permission table script to handle any RBAC resource

## Description

Using the script to generate the table for our RBAC resources currently fails because of unknown components:
- Remove hard coded list of sections/rules
  - Use `.metadata.name` directly as section with small modifications
  - Use same format as used in k8s documentation for rules (lowercase)



## How can this be tested?

1. Run `make doc/permissions`
2. Check contents of `permissions.md`
